sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/LogsIngestionAsyncClient.java

```diff
@@ -63,6 +87,12 @@ public final class LogsIngestionAsyncClient {
      * too large to be sent as a single request to the Azure Monitor service. In such cases, this method will split
      * the input logs into multiple smaller requests before sending to the service.
      *
+     * <p>
+     * Each log in the input collection must be a valid JSON object. The JSON object should match the
+     * <a href="https://learn.microsoft.com/azure/azure-monitor/essentials/data-collection-rule-structure#streamdeclarations">schema defined
+     * by the stream name</a>. The schema of the stream can be found in the Azure Portal.
```

Comment:
Update product name per the [style guide recommendation](https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=28008).
```suggestion
     * by the stream name</a>. The schema of the stream can be found in the Azure portal.
```